### PR TITLE
Update rubygem-foreman_bootdisk to 19.0.6

### PR DESCRIPTION
**Pull request by stejskalleos**
For Foreman 3.3

### CODE REVIEW

**Comment by stejskalleos** [comment]
@theforeman/packaging Hi can we merge the PR?

**Comment by evgeni** [comment]
Not if it's not building? ;)

File: packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec [hunk 1]
```diff
@@ -23,25 +23,23 @@ Requires:   /usr/bin/genisoimage
 
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.5
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: ruby >= 2.5
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec:26]: You're dropping SCL macros, but 3.3 us still building for EL7 with SCL

File: packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec [hunk 2]
```diff
@@ -23,25 +23,23 @@ Requires:   /usr/bin/genisoimage
...
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
-BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.15.0
-BuildRequires: %{?scl_prefix}npm(jed) >= 1.1.1
-BuildRequires: %{?scl_prefix}npm(jed) < 2.0.0
+BuildRequires: npm(@babel/core) >= 7.7.0
+BuildRequires: npm(@babel/core) < 8.0.0
+BuildRequires: npm(@theforeman/builder) >= 8.15.0
+BuildRequires: npm(@theforeman/builder) < 9.0.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec:40]: I bet this one can't be satisfied (we're using v10) and should not be here

**Review by evgeni — CHANGES_REQUESTED** [review]

**Pull request closed without merging**